src/main/java/com/enfint/dossierservice/config/KafkaTopics.java

```diff
@@ -0,0 +1,47 @@
+package com.enfint.dossierservice.config;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.config.TopicBuilder;
+
+@Configuration
+public class KafkaTopics {
```

Comment:
Not critical, but it will be better to define kafka topics in application.yaml file